tests/utils/test_metaprogramming.py

```diff
@@ -316,4 +326,25 @@ def test_context_manager():
             kind=ExecutableKind.IMPORT,
         ),
         "wraps": Executable(payload="from functools import wraps", kind=ExecutableKind.IMPORT),
+        "functools": Executable(payload="import functools", kind=ExecutableKind.IMPORT),
```

Comment:
Why do we even capture t his import?

Reply:
Because it's included in argument list of `fetch_data`'s decorator:

```
(Pdb) p decorator_vars(func, root_node=root_node)
['wraps', 'f', 'functools', 'WRAPPER_ASSIGNMENTS']
```

Due to [this](https://github.com/jd/tenacity/blob/0d40e76f7d06d631fb127e1ec58c8bd776e70d49/tenacity/__init__.py#L329) line, specifically.